readme.adoc

```diff
@@ -8,7 +8,7 @@ This work is licensed under a Creative Commons Attribution 4.0 International Lic
 
```

Comment:
It would be helpful to enhance with an explanation (or link) on how to use pre-commit.

Reply:
Agreed, I will submit a pr to the templates repo

Reply:
https://github.com/riscv/docs-spec-template/pull/32. I'll pull this in once the upstream PR has been accepted.